components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,120 @@
+.hods-timeline {
```

Comment:
To use the mixins I mentioned below you will need to copy this: https://github.com/UKHomeOffice/hods-poc/blob/master/components/status-banner/assets/StatusBanner.scss#L1

That will require you to `pnpm i @hods/sass-base` in the component's directory.